index.html

```diff
@@ -0,0 +1,209 @@
+                        </a>
+                    </li>
+                    <li>
+                        <a href="">
+                            <img src="images/twitter-circular-button.svg" alt="facebook-icon" style="background-color: rgba(255,230,38,0.7); height: 30px">
+                        </a>
+                    </li>
+
+                    <li>
+                        <a href="">
+                            <img src="images/linkedin-logo.svg" alt="facebook-icon" style="background-color: rgba(255,230,38,0.7); height: 30px">
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+        </figcaption>
+    </figure>
+    <header>
+        <h3 style="color: #ff0b68; padding-top: 3%; font-size: 250%">Sobre mim</h3>
+        <p style="color: #faffbd">Pat Peoples, um ex-professor de história na casa dos 30 anos, acaba de sair de uma instituição psiquiátrica.
+            Convencido de que passou apenas alguns meses naquele “lugar ruim”, Pat não se lembra do que o fez ir para lá.
+            O que sabe é que Nikki, sua esposa, quis que ficassem um "tempo separados". Tentando recompor o quebra-cabeças de sua memória,
+            agora repleta de lapsos, ele ainda precisa enfrentar uma realidade que não parece muito promissora. Com seu pai se recusando a
+            falar com ele, sua esposa negando-se a aceitar revê-lo e seus amigos evitando comentar o que aconteceu antes de sua internação,
+            Pat, agora um viciado em exercícios físicos, está determinado a reorganizar as coisas e reconquistar sua mulher,
+            porque acredita em finais felizes e no lado bom da vida. À medida que seu passado aos poucos ressurge em sua memória, Pat começa
+            a entender que "é melhor ser gentil que ter razão" e faz dessa convicção sua meta. </p>
+    </header>
+    <header>
+        <h2 style="color: #ffbf1d; text-align: left">Habilidades</h2>
+        <ul style="list-style-type: none">
+            <li style="border: 2px solid whitesmoke; padding: 3px; width: 25%; color: crimson; text-align: left">Javascript</li>
+            <li style="border: 2px solid whitesmoke; padding: 3px; width: 25%; color: crimson; text-align: left">Html e Css</li>
+            <li style="border: 2px solid whitesmoke; padding: 3px; width: 25%; color: crimson; text-align: left">JQuery</li>
+            <li style="border: 2px solid whitesmoke; padding: 3px; width: 25%; color: crimson; text-align: left">Wordpress</li>
+        </ul>
+    </header>
+</section>
+<section style="text-align: center">
+    <header>
+        <h3 style="color: #ff0b68; padding-top: 3%; font-size: 250%">Contato</h3>
+        <p style="color: #ffbf1d">Quentin Jacobsen tem uma paixão platônica pela magnífica vizinha e colega de escola Margo Roth Spiegelman.</p>
+        <p style="color: #ffbf1d">sestelos@gmail.com</p>
```

Comment:
Você pode facilitar a vida do teu usuário colocando seu email clicável.

```suggestion
        <a href="mailto:sestelos@gmail.com">sestelos@gmail.com</a>
```